android-components/components/support/remotesettings/src/main/java/mozilla/components/support/remotesettings/RemoteSettingsClient.kt

```diff
@@ -93,7 +75,26 @@ class RemoteSettingsClient(
     }
 
     /**
-     * Reads all records for a collection  found in the local storage.
+     * Updates the local storage with [records] data only if records changed from last time.
+     */
+    suspend fun writeLastModified(records: RemoteSettingsResponse, lastModified: ULong): RemoteSettingsResult =
```

Comment:
nit:
```suggestion
    suspend fun writeIfModifiedSince(records: RemoteSettingsResponse, lastModified: ULong): RemoteSettingsResult =
```
Let's make sure we have `@param` descriptions in the kdoc as well, since this is a public API

Reply:
I am kinda surprised detekt isn't flagging this 🤔